extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -422,8 +420,6 @@
 	window.ga('ads.set', 'dimension19', window.wgArticleType);                           // ArticleType
 	window.ga('ads.set', 'dimension20', 'not set');                                      // Performance A/B testing (not used any more)
 	window.ga('ads.set', 'dimension21', String(window.wgArticleId));                     // ArticleId
-	window.ga('ads.set', 'dimension23', window.wikiaIsPowerUserFrequent ? 'Yes' : 'No'); // IsPowerUser: Frequent
-	window.ga('ads.set', 'dimension24', window.wikiaIsPowerUserLifetime ? 'Yes' : 'No'); // IsPowerUser: Lifetime
```

Comment:
@nandy-andy - please code review this from AdEngineering / events tracking perspective

Reply:
Looks fine - I have never used this dimensions and removing them seems OK :)

Is there a way in GA to add a point in the timeline and inform us in future that until the day this pull request was merged dimensions 23 and 24 were connected to power users? ;)